.crews.php

```diff
@@ -315,16 +316,21 @@ function main () {
                 }
 
                 $positions = [
-                    'cc', 'driver', 'attendant', 'observer'
+                    'cc', 'driver', 'attendant', 'observer', 'dutysup'
                 ];
 
                 // Loops through all the spots and checks if there is a CC on in any of them.
                 foreach($positions as $pos) {
+                    // Duty sup is not actively on night crew
+                    if($pos == 'dutysup') {
+                        continue;
+                    }
+
                     $statement = $connection->prepare("SELECT * FROM members WHERE id = :memberid LIMIT 1");
-                    $statement->bindValue(':memberid', $y['cc']);
+                    $statement->bindValue(':memberid', $y[$pos]);
```

Comment:
This was only checking the CC crew slot, even though it was looping through each position